packages/e2e-tests/specs/editor/various/writing-flow.test.js

```diff
@@ -753,4 +753,42 @@ describe( 'Writing Flow', () => {
 		);
 		expect( paragraphs ).toEqual( [ 'first', 'second' ] );
 	} );
+
+	it( 'Allows block wrapper focus when there are no previous focus elements', async () => {
```

Comment:
This ensures Shift+Tab can focus the block wrapper if there are no previous elements. Then Shift+Tab will not work for anything before the block wrapper. Tab can move back in to the block.